includes/footer.php

```diff
@@ -5,17 +5,83 @@
                     <hr style="margin-top: -7px; ">
                 </div>
                 
-                <div class="row" class="sponsors">
-                    <div class="col-sm-4">
-                         <img class="img-responsive center-block" id="sponsor-kenworth" alt="Kenworth Logo" src="http://nctrucking.wildapricot.org/Resources/Pictures/Kenworth%20Bar%20Logo%20with%20A%20PACCAR%20Company%20Black%20Trans%20Background%20800%20DPIpng.png"/>
-                    </div>
-                    <div class="col-sm-4">                        
-                       <img class="img-responsive center-block" id="sponsor-SAE" alt="SAE Northwest Logo" src="http://www.sae.org/images/sections/ms049/NWL_380839334_sae_nw_logo_smaller.gif"/>
-                    </div>
-                    <div class="col-sm-4">
-                        <img class="img-responsive center-block" id="sponsor-boeing" alt="Boeing Logo" src="http://www.iptv.org/medialib/media/Boeing_RGB_blue_std_P.JPG"/>
+                <div id="carouselSponsor" class="carousel slide" data-ride="carousel" style="width: 400px; margin: 0 auto">
+                    <!-- Indicators -->
+                    <ol class="carousel-indicators">
+                        <li data-target="#carouselSponsor" data-slide-to="0" class="active"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="1"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="2"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="3"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="4"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="5"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="6"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="7"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="8"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="9"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="10"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="11"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="12"></li>
+                        
+                    </ol>
+
+                    <!-- Slides -->
+                    <div class="carousel-inner" role="listbox">
```

Comment:
Since we don't have much to put in the footer, it'd be nice if we could keep the 3 sponsor design on a viewport with appropriate width (i.e. laptop and desktop), but keep the one sponsor design on mobile (maybe with Bootstrap CSS tags?).